prerequisites/software.rst

```diff
@@ -309,3 +309,10 @@ An Elasticsearch plugin is required to index the contents of email attachments:
          Configuration and installation is out of our scope.
          Please follow the official vendor guides and ensure to have a
          tight security on your installation.
+
+2.10 Optional GnuPG for generating keys
+---------------------------------------------
+If you want to *generate* keys on your machine (for signing or encrypting mails), you need to install the GnuPG-Tools. 
+Please have a look at the official `GnuPG website`_. For the usage of keys, you don't have to install GnuPG (if you already have keys).
+
+.. _GnuPG website: https://www.gnupg.org/index.html
```

Comment:
GnuPG is needed to be installed for both usage and configuration of the PGP integration. We should reword this.

The link is also broken for me.